cmake/compiler_flags_NVHPC_Fortran.cmake

```
@@ -0,0 +1,13 @@
+# Precision-based Fortran compiler flags
+set(r8_flags "-r8") # Fortran flags for 64BIT precision
+
+# NVHPC Fortan
+set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} ")
+
+set(CMAKE_Fortran_FLAGS_RELEASE "-O3 -fast")
```

We are planning on simplifying our build type flags a bit, so we'd prefer to just have the optimization flag in there for more flexibility.
```suggestion
set(CMAKE_Fortran_FLAGS_RELEASE "-O3")
```